app/templates/main/browse-all.html

```diff
@@ -51,145 +55,135 @@
             </table>
         {% endif %}
     </div>
-    <!-- END TABLE -->
     <!-- FILTERS -->
     <div class="govuk-grid-column-one-third filters-form">
-        <div class="browse-filter-container">
-            <div class="browse-filter__header">
-                <p class="govuk-body-l browse__body filters-form__title">Filter within browse</p>
-                <img src="{{ url_for('static', filename='image/filter-icon.svg') }}"
-                     height="32px"
-                     width="32px"
-                     class="browse-filter__icon"
-                     alt="filter-icon">
-            </div>
-            <p class="govuk-body browse__body">Transferring body</p>
-            <div class="govuk-form-group filters-form__group filters-form__transferring-body-group">
-                <label class="govuk-label" for="browse_filter_transferring_body"></label>
-                <select class="govuk-select govuk-select__filters-form-transferring-body-select"
-                        id="browse_filter_transferring_body"
-                        name="sort">
-                    <option value="all" selected>Choose one...</option>
-                    <option value="arts">Arts Council England</option>
-                    <option value="food_standards_agency">Food Standards Agency</option>
-                    <option value="foreign_office">Foreign Office</option>
-                </select>
-            </div>
-        </div>
-        <div class="filters-form__series__container">
-            <p class="govuk-body browse__body">Series</p>
-            <div class="govuk-form-group filters-form__group filters-form__series-group">
-                <label class="govuk-label" for="browse_filter_series"></label>
-                <input class="govuk-input filters-form__series--input"
-                       id="browse_filter_series"
-                       name="width10"
-                       type="text">
+        <form method="post" action="{{ url_for('main.browse') }}">
+            {{ form.csrf_token }}
+            <div class="browse-filter-container">
+                <div class="browse-filter__header">
```

Comment:
whats the change here about form?

Reply:
I think I was working on the implementation of browse filters , that by mistake checked in , please let me know if you want me to remove

Reply:
removed the changes of  <form> tag not required  in browse_all.html page